packages/ng/button/button.component.ts

```diff
@@ -37,13 +39,39 @@ export class ButtonComponent implements OnChanges {
 	 */
 	luButton: '' | 'outlined' | 'text' | 'text-invert' = '';
 
+	#iconComponentRef?: ElementRef;
```

Comment:
```suggestion
	#iconComponentRef?: ElementRef<HTMLElement>;
```
not sure about this one